didcomm/pack_encrypted.py

```diff
@@ -119,12 +123,20 @@ async def pack_encrypted(
     # 5. protected sender ID if needed
     encrypt_res_protected = __protected_sender_id_if_needed(encrypt_res, pack_config)
 
-    # 6. do forward if needed
-    await __forward_if_needed()  # TBD
-
     packed_msg = dict_to_json(
         encrypt_res_protected.msg if encrypt_res_protected else encrypt_res.msg
     )
+
+    # 6. do forward if needed
+    #await __forward_if_needed(
+    packed_msg = await wrap_in_forward(
+        resolvers_config,
+        packed_msg,
+        to=to,
+        pack_config=pack_config,
+        pack_params=pack_params
+    )
+
     return PackEncryptedResult(
         packed_msg=packed_msg,
         service_metadata=ServiceMetadata("", ""),
```

Comment:
Let's not forget to return a ServiceMetadata here

Reply:
sure, thanks

Reply:
done